web/themes/custom/hatter_base/templates/navigation/menu--main.html.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Theme override to display a menu.
+ *
+ * Available variables:
+ * - menu_name: The machine name of the menu.
+ * - items: A nested list of menu items. Each menu item contains:
+ *   - attributes: HTML attributes for the menu item.
+ *   - below: The menu item child items.
+ *   - title: The menu link title.
+ *   - url: The menu link url, instance of \Drupal\Core\Url
+ *   - localized_options: Menu link localized options.
+ *   - is_expanded: TRUE if the link has visible children within the current
+ *     menu tree.
+ *   - is_collapsed: TRUE if the link has children within the current menu tree
+ *     that are not currently visible.
+ *   - in_active_trail: TRUE if the link is in the active trail.
+ */
+#}
+{% import _self as menus %}
+
+{#
+  We call a macro which calls itself to render the full tree.
+  @see http://twig.sensiolabs.org/doc/tags/macro.html
+#}
+{{ menus.menu_links(items, attributes, 0) }}
+
+{% macro menu_links(items, attributes, menu_level) %}
+  {% import _self as menus %}
+
+
+{% if menu_level == 0 %}
+<button class="primary-nav__trigger">{% include active_theme_path() ~ '/svg/menu.svg' %} Menu</button>
```

Comment:
That is correct - this base theme is not intended to be used on its own. It mainly exists so that we can share existing templates easily. The themes that use it can override templates as necessary, and also require a specific release of hatter (if we want to keep other themes alive for theme switching for example.) I'll add a note about this in the base theme.